main/src/ca/uwaterloo/flix/language/phase/Resolver.scala

```diff
@@ -1481,4 +1482,65 @@ object Resolver extends Phase[NamedAst.Root, ResolvedAst.Root] {
     Name.NName(sp1, idents, sp2)
   }
 
+  /**
+   * Creates the synthetic type classes `Eq` and `Ord`.
+   */
+  private def mkSynthClasses()(implicit flix: Flix): Map[Symbol.ClassSym, ResolvedAst.Class] = {
+    val eqTypeClass = mkEqClass()
+    val ordTypeClass = mkOrdClass(eqTypeClass)
```

Comment:
We can also add: `Show`.

Reply:
And `Hash`

Reply:
I think we should pick one type class -- probably `Show` and use that to get the implementation working.

Messing with `Eq` and `Hash` while not everything is working will be cumbersome, but we can use `Show`.